packages/astro/src/integrations/features-validation.ts

```diff
@@ -124,20 +124,20 @@ function validateSupportKind(
 }
 
 function featureIsUnsupported(adapterName: string, logger: Logger, featureName: string) {
-	logger.error('config', `The feature "${featureName}" is not supported (used by ${adapterName}).`);
+	logger.error('config', `The adapter ${adapterName} doesn't support the feature "${featureName}" yet.`);
```

Comment:
```suggestion
	logger.error('config', `The adapter ${adapterName} doesn't currently support the feature "${featureName}".`);
```
This implies that it *will* support the feature, and we might not want to make that promise. I think it's fine just to say that it doesn't currently support it!

Reply:
Good call!